vehicles/src/modules/permit/permit.service.ts

```diff
@@ -259,7 +249,7 @@ export class PermitService {
       });
     }
 
-    if (expired?.toLowerCase() === 'true') {
+    if (!getPermitQueryParamsDto.expired) {
       permitsQuery = permitsQuery.andWhere(
         '(permit.permitStatus IN (:...expiredStatus)OR(permit.permitStatus = :activeStatus AND permitData.expiryDate < :expiryDate))',
```

Comment:
Is there supposed to be a space between the 'OR' and the two brackets ) and (

Reply:
yes, however, typeorm and mssql drivers were somehow intelligent enough to resolve it at runtime. I will fix it anyway.

The PR has been tagged 'Don't Review', 'Don't Merge'

Reply:
ok I see, np